src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -439,22 +439,24 @@ Path getLlcPath() throws IOException {
         return answer;
     }
 
-    private static final List<String> resourcesList = Arrays.asList(
+    private static final List<String> RESOURCES_BY_EXTENSION = Arrays.asList(
             "frag", "fxml", "css", "gls", "ttf", "xml",
             "png", "jpg", "jpeg", "gif", "bmp",
             "license", "json");
 
-    private  List<String> getResources() {
-        List<String> resources = new ArrayList<>(resourcesList);
-        resources.addAll(projectConfiguration.getResourcesList());
+    private List<String> getIncludeResourcesArguments() {
+        List<String> resourcesByExtension = RESOURCES_BY_EXTENSION.stream()
+                .map(extension -> "-H:IncludeResources=.*\\." + extension + "$")
+                .collect(Collectors.toList());
 
-        List<String> list = resources.stream()
-                .map(s -> "-H:IncludeResources=.*/.*" + s + "$")
+        List<String> configurationResources = projectConfiguration.getResourcesList().stream()
+                .map(resource -> "-H:IncludeResources=" + resource)
```

Comment:
This implies the user will need to provide a regex too? I'm okay with this, as it will be more flexible probably, but we'll need to update the docs about it.

Reply:
It's indeed more flexible, and the docs currently say the following:

> List of additional resource patterns or extensions that will be added...

It might be an improvement to explicitly mention that it should be a regular expression pattern.

Reply:
We should do a follow-up on this, even adding a small example to the docs.